### PR TITLE
French translations

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Added French translations to the best of my skills! @anikbrazeau, I'm happy to change a few things, here are some screenshots.

## First page
Screen Shot 2020-10-30 at 15 33 48

## Second page
Screen Shot 2020-10-30 at 15 33 56

### CODE REVIEW

File: tests/app/main/views/test_set_lang.py
Comment (brdunfield) [inline tests/app/main/views/test_set_lang.py:1]: It doesn't look to me like any tests are being run in CI in this repo. That's not an issue with this PR specifically, but would be nice to fix.

**Review by brdunfield — APPROVED** [review]
LGTM on the code side - tests pass locally.

I'd wait for Anik's ok before merge! 🦑

**Comment by anikbrazeau** [comment]
Looks good. Could we make one little tweak?

EN Try again later.
FR **Réessayez** plus tard.

File: tests/app/main/views/test_set_lang.py
Comment (AntoineAugusti) [reply tests/app/main/views/test_set_lang.py:1]: Added CI in https://github.com/cds-snc/notification-document-download-frontend/pull/30 and I'll rebase

**Pull request merged**
